pkgs/top-level/all-packages.nix

```diff
@@ -429,8 +425,6 @@ with pkgs;
   };
 
   cereal = cereal_1_3_0;
```

Comment:
Move this to `aliases.nix` maybe?

Reply:
Same as below

Reply:
This is a "green alias", because some packages don't work with the newer version.